ext/standard/css.c

```diff
@@ -31,7 +31,6 @@ PHPAPI ZEND_COLD void php_info_print_css(void) /* {{{ */
 	PUTS("th {position: sticky; top: 0; background: inherit;}\n");
 	PUTS("h1 {font-size: 150%;}\n");
 	PUTS("h2 {font-size: 125%;}\n");
-	PUTS("h2 a:link, h2 a:visited{color: inherit; background: inherit;}\n");
```

Comment:
```diff
- PUTS("h2 a:link, h2 a:visited{color: inherit; background: inherit;}\n");
+ PUTS("h2 > a {text-decoration: none;}\n");
+ PUTS("h2 > a:hover {text-decoration: underline;}\n");
```

Reply:
While using the global `<a>` tag CSS to underline the link, could we add an override for the anchor links so we show them without the underline?

My last CSS change was to add anchor links to each extension config section. That CSS was so the title continues to look as before, but are now clickable.

Reply:
Normal state:
![image](https://github.com/user-attachments/assets/9b618ee0-8a65-4e0c-8e18-1d76f1146e1c)

Hover state:
![image](https://github.com/user-attachments/assets/c9422857-5a11-46ff-97ab-74bf81310826)

Reply:
I have considered preserving that, but found that the underline did not look too shabby, so I went for that for the initial proposal.

I've now adjusted this to your suggestion.

Reply:
Awesome, thank you!